src/frontends/jax/src/op/select_n.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/jax/node_context.hpp";
+#include "openvino/op/ops.hpp";
```

Comment:
please include concrete operations headers instead. That is because "ops.hpp" contains many-many includes for each operation that slow downs compilation and requires more resources on build machine.

Reply:
I have added headers that are required